test/mpi/f90/f90types/createf90types.c

```diff
@@ -51,38 +51,38 @@ static int checkType(const char str[], int p, int r, int f90kind, int err, MPI_D
             errs++;
             printf("Wrong combiner type (got %d, should be %d) for %s\n", combiner, f90kind, str);
         } else {
-            int parms[2];
+            int params[2];
```

Comment:
this doesn't seem right.

Reply:
I'll rename it to `ints` since we use `ints` in everywhere else.